duckduckgo/privacy-dashboard
# Toggle Report translations + minor fixes

Asana: https://app.asana.com/0/72649045549333/1208865759492772/f
**Reviewer:** @shakyShane 

## Description:

1. Updated translations for toggle report data disclosure fields “locale”, “description” and “features"

**Note: due to the list being dynamic and not being able to guarantee which data disclosure field will appear last, the translator suggested removing semi-colons and periods from the end of list items on Russian (and look at reapplying them with CSS). Which explains why the Russian diff is longer**

2. Fixed top-bottom padding of Toggle Report screen on iOS

## Steps to test this PR:

1. Visit the [Polish](https://deploy-preview-253--ddgdash.netlify.app/app-debug/html/iframe?state=locale-pl&screen=toggleReport) version of the toggle report
2. Click on the data disclosure link (Zobacz, co wysłano)
3. Scroll to the bottom of the list and ensure you see the translated versions of "locale" (Główny język i kraj urządzenia) and "description" (Wybrana kategoria i opcjonalne komentarze)
